src/components/nodes/boundaryMessageEvent/boundaryMessageEvent.vue

```diff
@@ -11,7 +12,7 @@ export default {
     },
   },
   mounted() {
-    const shapeAttributes = getIntermediateMailIconShapeAttributes();
+    const shapeAttributes = getIntermediateMailIconShapeAttributes('#000', 1, '#fff', boundaryMessageMailSymbol);
```

Comment:
Because you've added an argument that requires a value after ones that have defaults this call is now required to pass all arguments.